en/02_Developer_Guides/00_Model/02_Relations.md

```diff
@@ -989,9 +989,45 @@ class ParentObject extends DataObject
 }
 ```
 
-In this example, when the parent object is deleted, the child specified by the `has_one` relation will also
-be deleted. Note that all relation types (`has_many`, `many_many`, `belongs_many_many`, `belongs_to`, and `has_one`)
-are supported, as are methods that return lists of objects but do not correspond to a physical database relation.
+In this example, when the parent object is deleted, the child specified by the `has_one` relation will also be deleted. The relation types `has_many`, `belongs_to`, and `has_one` are supported, as are methods that return lists of objects but do not correspond to a physical database relation.
+
+### Cascading deletions with `has_many`
```

Comment:
This says `has_many` but the example shows a `many_many`... that's a bit confusing. This should probably be
```suggestion
### Cascading deletions for `many_many` relations
```
or perhaps
```suggestion
### Cascading deletions for `many_many` through relations
```

And then explain what using `cascade_deletes` directly on the `many_many` relation does, and provide your example as an alternative and describe what the behaviour is for that and how it differs.

Reply:
🤦